.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -135,51 +127,15 @@ jobs:
           python3.11 -m pip install packaging wheel setuptools-scm
           python3.11 -m pip install .[cuda]
 
-      - name: ilab system info
-        run: |
-          . venv/bin/activate
-          ilab system info
-
       - name: Run e2e test
         run: |
-          nvidia-smi
-          # This env variable is used on GPUs with less vRAM. It only allows cuda to alloc small chunks of vRAM at a time, usually helping to avoid OOM errors.
-          # This is not a good solution for production code, as setting ENV variables for users isn't best practice. However, it is a helpful manual workaround.
-          export PYTORCH_CUDA_ALLOC_CONF=garbage_collection_threshold:0.6,max_split_size_mb:128
           . venv/bin/activate
-          # TODO: for some reason we need to reinstall DS in order to get fused adam support
-          # This means we need to manually rm and re-install a bunch of packages. Investigate why this is.
-          python3.11 -m pip uninstall -y deepspeed
-
-          python3.11 -m pip cache purge
+          ./scripts/e2e.sh -m
 
-          DS_BUILD_CPU_ADAM=1 BUILD_UTILS=1 python3.11 -m pip install deepspeed
```

Comment:
This is sort of tribal knowledge we should keep track of for the a10x4 if not already there

Reply:
What do you mean?

Reply:
This might be worth documenting in ci.md that in any runner using NVIDIA GPUs, we manually configure the machine apart from the ami we use.

Reply:
Oh I see - so related to your comment above?

I def agree we need some kind of documenting around these configs, though my preference would be having things in the README and the CI config is informed by the README

Reply:
Discussed more in-person today